### ISSUE TITLE
Upgrade to zimit 0.12

### ISSUE THREAD

**Issue by benoit74** [openzim/zimit#231]
This is not a real issue, more a placeholder to list known impacts that have been identified so far.

**Comment by benoit74** [comment]
While moving to zimit 0.12, Chromium is replaced by Brave.

This obviously has an impact on WARCZ content.

On test site "isago.rskg.org", we previously has 9 items in the WARCZ:
```
[DEBUG] Adding https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css
[DEBUG] Adding https://isago.rskg.org/
[DEBUG] Adding https://isago.rskg.org/a-propos
[DEBUG] Adding https://isago.rskg.org/conseils
[DEBUG] Adding https://isago.rskg.org/faq
[DEBUG] Adding https://isago.rskg.org/static/favicon256.png
[DEBUG] Adding https://isago.rskg.org/static/tarifs-isago.pdf
[DEBUG] Adding https://content-autofill.googleapis.com/v1/pages/ChRDaHJvbWUvMTEyLjAuNTYxNS40ORIeCXjz9Q1zsMP8EgUNnpR2qRIFDdOVJAISBQ0MdULB?alt=proto
[DEBUG] Adding https://accounts.google.com/ListAccounts?gpsia=1&source=ChromiumBrowser&json=standard&__wb_method=POST& 
```

There are now 10 items:
```
[DEBUG] Adding https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css
[DEBUG] Adding https://isago.rskg.org/
[DEBUG] Adding https://isago.rskg.org/a-propos
[DEBUG] Adding https://isago.rskg.org/conseils
[DEBUG] Adding https://isago.rskg.org/faq
[DEBUG] Adding https://isago.rskg.org/static/favicon256.png
[DEBUG] Adding https://isago.rskg.org/static/tarifs-isago.pdf
[DEBUG] Adding https://laptop-updates.brave.com/1/usage/brave-core-threshold?platform=linux-bc&channel=release&version=1.59.120&daily=true&weekly=true&monthly=true&first=true&woi=2023-10-30&dtoi=null&ref=BRV001&adsEnabled=false&arch=&wallet2=0
[DEBUG] Adding https://laptop-updates.brave.com/1/usage/brave-core?platform=linux-bc&channel=release&version=1.59.120&daily=true&weekly=true&monthly=true&first=true&woi=2023-10-30&dtoi=null&ref=BRV001&adsEnabled=false&arch=&wallet2=0
[DEBUG] Adding https://dict.brave.com/edgedl/chrome/dict/en-us-10-1.bdic
```

Looks like the difference is the items which are automatically retrieved by the Browser, no impact expected for us.

**Comment by benoit74** [comment]
0.12 has been released, we packaged it in 1.6.0
